packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -68,13 +81,16 @@ const trace = makeTracer('VM', false);
  * @property {Amount<'nat'>}  totalCollateral    present sum of collateral across all vaults
  * @property {Amount<'nat'>}  totalDebt          present sum of debt across all vaults
  * @property {Amount<'nat'>}  retainedCollateral collateral held as a result of not returning excess refunds
- *                                                from AMM to owners of vaults liquidated with shortfalls
+ *                                                to owners of vaults liquidated with shortfalls
+ * @property {Amount<'nat'>}  liquidatingCollateral  present sum of collateral in vaults sent for liquidation
+ * @property {Amount<'nat'>}  liquidatingDebt        present sum of debt in vaults sent for liquidation
  *
  * @property {Amount<'nat'>}  totalCollateralSold       running sum of collateral sold in liquidation
  * @property {Amount<'nat'>}  totalOverageReceived      running sum of overages, central received greater than debt
- * @property {Amount<'nat'>}  totalProceedsReceived     running sum of central received from liquidation
- * @property {Amount<'nat'>}  totalShortfallReceived    running sum of shortfalls, central received less than debt
- * @property {number}         numLiquidationsCompleted  running count of liquidations
+ * @property {Amount<'nat'>}  totalProceedsReceived     running sum of minted received from liquidation
+ * @property {Amount<'nat'>}  totalShortfallReceived    running sum of shortfalls, minted received less than debt
+ * @property {number}         numLiquidationsCompleted  running count of liquidated vaults
+ * @property {number}         numVaultsReconstituted    running count of vaults reconstituted after unsuccessful liquidation
```

Comment:
should this be `numLiquidations…`? if one vault is liquidated and reverted twice, does the this counter go up by one or two? (if two then it definitely shouldn't be named "numVaults…")

Reply:
agreed on `numLiquidationsAborted`